build-support/bin/release.sh

```diff
@@ -261,9 +264,10 @@ function build_fs_util() {
 }
 
 function activate_tmp_venv() {
+  # shellcheck source=/dev/null
```

Comment:
May be worth explaining the /dev/null sources in a comment

Reply:
Agreed.